2025-03-04 10:50:06,667 INFO    MainThread:1682172 [internal.py:wandb_internal():90] W&B internal server running at pid: 1682172, started at: 2025-03-04 10:50:06.667321
2025-03-04 10:50:06,668 INFO    WriterThread:1682172 [datastore.py:open_for_write():75] open: /media/isaac/Daten/azhar_ws/OmniIsaacGymEnvs/omniisaacgymenvs/scripts/wandb/run-20250304_105006-AnymalTerrain_2025-03-04_10-50-05/run-AnymalTerrain_2025-03-04_10-50-05.wandb
2025-03-04 10:50:06,807 DEBUG   SenderThread:1682172 [sender.py:send():236] send: header
2025-03-04 10:50:06,807 DEBUG   SenderThread:1682172 [sender.py:send():236] send: run
2025-03-04 10:50:06,824 INFO    SenderThread:1682172 [sender.py:_maybe_setup_resume():493] checking resume status for bluewolf/rl/AnymalTerrain_2025-03-04_10-50-05
2025-03-04 10:50:07,443 INFO    SenderThread:1682172 [dir_watcher.py:__init__():216] watching files in: /media/isaac/Daten/azhar_ws/OmniIsaacGymEnvs/omniisaacgymenvs/scripts/wandb/run-20250304_105006-AnymalTerrain_2025-03-04_10-50-05/files
2025-03-04 10:50:07,443 INFO    SenderThread:1682172 [sender.py:_start_run_threads():815] run started: AnymalTerrain_2025-03-04_10-50-05 with start time 1741081806
2025-03-04 10:50:07,446 DEBUG   SenderThread:1682172 [sender.py:send():236] send: summary
2025-03-04 10:50:07,447 INFO    SenderThread:1682172 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2025-03-04 10:50:07,449 DEBUG   HandlerThread:1682172 [handler.py:handle_request():140] handle_request: check_version
2025-03-04 10:50:07,450 DEBUG   SenderThread:1682172 [sender.py:send_request():250] send_request: check_version
2025-03-04 10:50:07,500 WARNING MainThread:1682172 [internal.py:wandb_internal():153] Internal process interrupt: 1
2025-03-04 10:50:08,000 WARNING MainThread:1682172 [internal.py:wandb_internal():153] Internal process interrupt: 2
2025-03-04 10:50:08,000 ERROR   MainThread:1682172 [internal.py:wandb_internal():156] Internal process interrupted.
2025-03-04 10:50:08,049 INFO    MainThread:1682172 [internal.py:handle_exit():80] Internal process exited
